### ISSUE TITLE
Export Artifacts issue

### ISSUE THREAD

**Issue by markhu520** [osrmt/osrmt#57]
Package:osrmt_1.8_desktop
After add feature, requirement, design and testcase information.
Click File--->Export, no export button display on the popup window.

![export issue1](https://user-images.githubusercontent.com/72968518/103495571-38252d00-4e76-11eb-86d5-51e9905d0ca8.PNG)

**Comment by TS7-development** [comment]
Same for me, sadly... the tool was looking so promising...

**Comment by ZoranPandovski** [comment]
Hey @TS7-development, this project has not been maintained for a long time. Feel free to make a PR in case you solve the above issue :raised_hands:

**Comment by TS7-development** [comment]
Yes I have seen that, it was not maintained for a long time. The comment had more a purpose in documenting, that this is still the case, in case someone else is trying it. Should I look into the code and fix the problem, I will make a PR @ZoranPandovski.
